doc/Language/faq.pod6

```diff
@@ -194,7 +194,7 @@ L«C<App::Mi6> tool|https://modules.raku.org/l/App::Mi6» can simplify the
 uploading process.  The
 L«C<zef> module installer|https://github.com/ugexe/zef»
 automatically check for latest versions of a module on CPAN as well as our
-L<GitHub-based ecosystem|https://github.com/perl6/ecosystem/>.
+L<GitHub-based ecosystem|https://github.com/raku/ecosystem/>.
```

Comment:
Canonical URL uses caps: Raku/ecosystem. It works this way, but it's better to use the canonical one.